tests/README.md

```diff
@@ -31,3 +31,25 @@ Installs all the required mvn dependencies.
 ### `mvn test`
 
 Runs the tests on the specified browser.
+
+## Running the Cucumber Tests with tags
+Add values to the keys in config.properties file
+Update tags value in 'RunCucumberTest' class
+
+In the 'tests' directory, you can run:
+
+### `mvn test -Dcucumber.options="--tags '@tagname'"`
+
+This runs the tests that are tagged with the specific tag name. 
+Without the 'tags' option all tests gets executed.
+
+Alternatively, both frontend and backend tests can be run from 
+
+### `RunCucumberTest.java` class
```

Comment:
I think these shouldn't be headings...? Alex might want to chime in. We were talking about readme formatting earlier in the day.

Reply:
Fixed. Thanks!